app/javascript/components/support/SupportPortal.tsx

```diff
@@ -18,7 +18,7 @@ export default function SupportPortal() {
     const url = new URL(location.href);
     if (!isNewTicketOpen && url.searchParams.get("new_ticket")) {
       url.searchParams.delete("new_ticket");
-      router.get(url.toString(), { replace: true, preserveState: true, preserveScroll: true });
+      router.get(url.toString(), {}, { viewTransition: true });
```

Comment:
`replace: true, preserveState: true, preserveScroll: true` should not be removed but moved to the options object in lieu of viewTransition

Reply:
I think the original code before inertia migration just wanted to clear the query param on close? So maybe we don't need those other options. Can you please confirm this?

https://github.com/user-attachments/assets/666c2656-b005-46c7-8cc1-31f544657931